src/test/extensions/updateHelper.spec.ts

```diff
@@ -563,3 +581,71 @@ describe("inferUpdateSource", () => {
     expect(result).to.equal("notfirebase/storage-resize-images@latest");
   });
 });
+
+describe("getExistingSourceOrigin", () => {
+  let registryEntryStub: sinon.SinonStub;
+  let isOfficialStub: sinon.SinonStub;
+  let getInstanceStub: sinon.SinonStub;
+
+  afterEach(() => {
+    registryEntryStub.restore();
+    isOfficialStub.restore();
+    getInstanceStub.restore();
+  });
+
+  it("should return official extension as source origin", async () => {
+    registryEntryStub = sinon.stub(resolveSource, "resolveRegistryEntry");
+    registryEntryStub.resolves(REGISTRY_ENTRY);
+    isOfficialStub = sinon.stub(resolveSource, "isOfficialSource");
+    isOfficialStub.returns(true);
+    getInstanceStub = sinon.stub(extensionsApi, "getInstance").resolves(INSTANCE);
+    const result = await updateHelper.getExistingSourceOrigin(
```

Comment:
Nit: Repo convention is to split unit tests into 3 chunks separated by newlines- setup, call functions under test, and expects. For example, in this test, you should add a newline between l601 and l602, and one between l607 and l608.

Reply:
Sounds good. Thanks for the heads up!